### DISCUSSION TITLE
24,000kHz

### DISCUSSION THREAD

**Discussion by ghost** [coqui-ai/TTS#3682] [General Q&A]
Why does AI reduce the sound quality from 41,000kHz to 24,000kHz?Because of this, the generated speech sounds almost like on a walkie-talkie. Is it really impossible to force her to save 41,000kHz

**Comment by ghost** [comment]
A 24000 high-frequency slice can be restored in FL studio using an equalizer.

**Reply by identificator** [reply]
You can't improve 24k audio output quality to 44.1k with EQ, it's physically impossible.

**Reply by ghost** [reply]
Yes, it's not the real 44100, but it's better than the real 24000
[24000 vs 44100.zip](https://github.com/coqui-ai/TTS/files/15299598/24000.vs.44100.zip)
